processes/security_team_members.md

```diff
@@ -98,3 +108,5 @@ who have access to
 
 Every member of the team with access to security issues should have access to
 the private security patches as well.
+
+XXX ... But they don't, and if they did, we wouldn't have to put the list in twice.
```

Comment:
I'm not sure what you mean here. If you are in @nodejs-private/security, you have access. (Or am I misunderstanding how something works?)

IIUC, @nodejs/security is basically legacy except that it's useful for @-notifications in repos in the @nodejs org.

Reply:
There are two sections here, one above for "issues" and this one for "patches". First is doced as being the people in @nodejs/security, the second is doced as org/nodejs-private/people... but then it says everyone in @nodejs/security should be in nodejs-private/people... so they are basically the same lists.  And also, the "issues" is https://github.com/nodejs-private/security and the patches are https://github.com/nodejs-private/node-private, and it happens that BOTH issues and patches have exactly the same team: https://github.com/orgs/nodejs-private/teams/security. All of which is to say, describing them as having two different lists of members is not correct.

Also, the issues have been archived... so its not even relevant. Anyhow, these are more notes for myself, I'll do a second pass and clean this up.